### PR TITLE
51 create rrd for checks

### PR DESCRIPTION
**Pull request by hutsulyk**
Create RRD Tool wrapper to use it with checks :chart_with_upwards_trend: 

Use these methods in `check_launcher` to update check RRDs :slightly_smiling_face:

### CODE REVIEW

**Review by lisovyk — APPROVED** [review]
Well done!

**Review by ghost — APPROVED** [review]
good job!

File: moninag/daemons/check_launcher.py [hunk 1]
```diff
@@ -84,6 +85,10 @@ def status_change(check_id, output):
     this_check.last_run = timezone.now()
     this_check.save()
     this_check.update_service_status()
+
+    # Update check RRD
+    check_rrd.update(this_check.id, this_check.output)
+
     time.sleep(1)
```
Comment (lhalam) [inline moninag/daemons/check_launcher.py]: the magic number, move to default argument

File: moninag/performance/check_rrd.py [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+
+def generate_perfgraphs(check_id):
+    """Generate check graphs for last hour, day, week, month"""
+
+    hour_graph_dir = CHECKS_GRAPH_PATH + 'hour/'
+    day_graph_dir = CHECKS_GRAPH_PATH + 'day/'
+    week_graph_dir = CHECKS_GRAPH_PATH + 'week/'
+    month_graph_dir = CHECKS_GRAPH_PATH + 'month/'
+
+    generate_graph(check_id, hour_graph_dir, time_period=3600)
```
Comment (lhalam) [inline moninag/performance/check_rrd.py]:
```
3600=>60*60
86400=> 60*60*24
...
```

File: moninag/performance/check_rrd.py [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+
+
+def generate_graph(check_id, graph_dir, time_period=3600, width=800, height=300):
+    """Generate check graph"""
+
+    rrd_name = _generate_name(CHECKS_RRD_PATH, check_id, 'rrd')
+
+    if not exist(rrd_name):
+        LOGGER.error('GRAPH RRD: check#{0}, FAIL: RRD does not exist.'.format(check_id))
+        return
```
Comment (lhalam) [inline moninag/performance/check_rrd.py]: return None

**Review by ghost — APPROVED** [review]
GJ 👏

**Review by lhalam — APPROVED** [review]

**Pull request merged**